package.json

```diff
@@ -28,13 +28,14 @@
     "lint:tests": "eslint ./tests/** --ignore-path .gitignore --fix",
     "lint:fix": "npm run lint -- --fix",
     "formatlib": "find warplib/ -iname *.cairo -exec cairo-format -i {} +",
-    "test": "tests/behaviour/setup.sh && npx mocha \"tests/**/*.test.ts\" --extension ts --require ts-node/register --parallel --exit --exclude tests/cli/**/*.ts",
+    "test": "tests/behaviour/setup.sh && npx mocha \"tests/**/*.test.ts\" --extension ts --require ts-node/register --parallel --exit --exclude tests/cli/**/*.ts --exclude tests/compilation/**/*.ts",
     "test:cli": "npx mocha tests/cli/**/*.test.ts --extension ts --require ts-node/register --exit",
     "test:forwarder": "npx mocha tests/interfaceCallForwarder/interfaceForwarder.test.ts --extension ts --require ts-node/register --exit",
     "test:lib": "npx mocha tests/warplib.test.ts --extension ts --require ts-node/register --exit",
     "test:behaviour": "npx mocha tests/behaviour/behaviour.test.ts --extension ts --require ts-node/register --exit",
     "test:semantic": "tests/behaviour/setup.sh && node build/semanticTestRunner.js",
-    "test:examples": "npx mocha exampleContracts/exampleContracts.test.ts --extension ts --require ts-node/register --exit",
+    "test:examples": "yarn tsc; npx mocha exampleContracts/exampleContracts.test.ts --extension ts --require ts-node/register --exit",
```

Comment:
Why are we compiling the project (`yarn tsc`) before actually running the tests inside the `test` command? All of the other `test` commands don't have it and I agree that this is a separate step. In the workflow, once the dependencies are installed the project is compiled so it will be doing this step at least twice.